src/LeanCode.ContractsGenerator/Analyzers/ErrorCodesUniqueness.cs

```diff
@@ -4,9 +4,9 @@ public class ErrorCodesUniqueness : BaseAnalyzer
 {
     public const string Code = "CNTR0003";
 
-    public override IEnumerable<AnalyzeError> AnalyzeErrorCodes(AnalyzerContext context, IEnumerable<ErrorCode> errorCodes)
+    public override IEnumerable<AnalyzeError> AnalyzeErrorCodes(AnalyzerContext context, IEnumerable<ErrorCode> errCodes)
```

Comment:
Why the change?

Reply:
Analyzers complain when the name used in the override is different from the one used in interface / base class.